### ISSUE TITLE
Bisq price discrepance

### ISSUE THREAD

**Issue by lorenzoPrimi** [vicariousdrama/nodeyez#122]
At the moment I get this bitcoin price on my panels: **53549** USD
while the price on coingeko is **51052** USD
This is reflected in the bisq api:
```
"btc_usd": { 
"last": "53549.20120000",
"high": "66290.82280000",
"low": "48197.55880000",
"volume_left": "0.77620000",
"volume_right": "40181.41710000",
"buy": null,
"sell": null
}
```

Is there a way to use another, more reliable source of price without modifying too much code?

**Comment by vicariousdrama** [comment]
yes there are other APIs that this could be retrofitted against as an option. The kraken API provides useful data, but they all need normalized
